[DOCS] Expands anomaly detection alert type docs (#70026)
elastic/elasticsearch: docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

== Creating an alert

You can create {anomaly-detect} alerts in the {anomaly-job} wizard after you 
start the job, from the job list, or under **{stack-manage-app} > {alerts-ui}**. 
On the *Create alert* window, select *{anomaly-detect-cap} alert* under the 
{ml-cap} section, then give a name to the alert and optionally provide tags.

Specify the time interval for the alert to check detected anomalies. It is 
recommended to select an interval that is close to the bucket span of the 
associated job. You can also select a notification option by using the _Notify_ 
selector. An alert instance remains active as long as anomalies are found for a 
particular {anomaly-job} during the check interval. When there is no anomaly 

> found in the next interval, the state of the alert instance changes to 
> `Recovered`, then transitions to `OK`. For more details, refer to the

Reviewer: to alight with the alerting framework terminology, It should be `status`, not `state`. Also, `Recovered` is a built-in action group.

```suggestion
found in the next interval, the `Recovered` action group is invoked and the status of the alert instance changes to `OK`. For more details, refer to the
```

previous anomalies. The default severity threshold is 75 which means every 
anomaly with an `anomaly_score` of 75 or higher triggers the alert.

You can select whether you want the alert to include interim results. Interim 
results are created by the {anomaly-job} before a bucket is finalized. These 

> results might disappear after the bucket is fully processed. Allow including 

Reviewer 2: Unless "allow" is part of the UI text, it seems unnecessary here:
```suggestion
results might disappear after the bucket is fully processed. Include 
```

interim results if you want to be notified earlier about a potential anomaly 

> even if it might be a false positive. Disallow them if you want to get notified 

Reviewer 2: ```suggestion
even if it might be a false positive. If you want to get notified 
```

> only about anomalies of fully processed buckets. 

Reviewer 2: ```suggestion
only about anomalies of fully processed buckets, do not include interim results.
```

[role="screenshot"]
image::images/ml-anomaly-alert-messages.jpg["Customizing your message"]

After you save the configurations, the alert appears in the *{alerts-ui}* list 
where you can check its status and see the overview of its configuration 
information.

The name of an alert instance is always the same as the job ID of the associated 

> {anomaly-job}. You can mute the notifications for a particular {anomaly-job} on 

Reviewer: The user can assign multiple anomaly detection jobs, hence we should adjust this sentence slightly

After you save the configurations, the alert appears in the _Alerts and 
Actions_ list where you can check its status and see the overview of its 
configuration information.

The name of an alert instance is always the same as the job ID of the associated 
{anomaly-job}. You can mute the notifications for a particular {anomaly-job} on 
the page of the alert that lists the individual alert instances. You can open it 

> via {alerts-ui} by selecting the alert name.

Reviewer 2: > {alerts-ui} by selecting the alert name.

There's an occurrence of this "Alerts and Actions" in the previous paragraph too, though (1) it doesn't use this shared attribute and (2) it's italicized.  I think both should use the shared attribute, and both should be bolded.
